airflow/sensors/base.py

```diff
@@ -114,15 +118,33 @@ def _orig_start_date(
     )
 
 
+class SkipPolicy(str, enum.Enum):
+    """Class with sensor's skip policies."""
+
+    # if poke method raise an exception, sensor will not be skipped on.
+    NONE = "none"
+
+    # If poke method raises an exception, sensor will be skipped on.
+    SKIP_ON_ANY_ERROR = "skip_on_any_error"
+
+    # If poke method raises AirflowSensorTimeout, AirflowTaskTimeout, AirflowFailException
+    # sensor will be skipped on.
+    SKIP_ON_SOFT_ERROR = "skip_on_soft_error"
```

Comment:
I might suggest rewording this one as well. The description here is great, but `soft`_fail sounds a bit unclear to me

Reply:
ok , what is your proposition ?

Reply:
Hi, sorry for the late reply. Nothing comes to my mind as of now. As long as we document it, I think we're good. Thanks 🙂 Resolve this conversation.

Reply:
what about SKIP_ON_TIMEOUT and remove AirflowFailException from the list of exception catch by option

naming would be explicit and correct

wdyt ?